test/int/adapter.test.ts

```diff
@@ -15,8 +15,9 @@ import { isWindows } from './testSetup';
 import * as puppeteer from 'puppeteer';
 import { expect } from 'chai';
 import { killAllChrome } from '../testUtils';
-import { IAttachRequestArgs } from 'vscode-chrome-debug-core';
+import { IAttachRequestArgs, utils } from 'vscode-chrome-debug-core';
 import { getDebugAdapterLogFilePath } from './utils/logging';
+import { DebugClient } from 'vscode-debugadapter-testsupport';
```

Comment:
It might be better to rebase the tests from v1. If you add them we are going to have more conflicts later...

Reply:
I think we chatted about this on an earlier PR, but I can't remember the solution we came to. We can't rebase the whole branch because we'd mess up the history for all of V2, and we can't rebase _just_ the tests.

Reply:
You are right. When I say re-base I actually mean merge...

Reply:
done